plugins/omkafka/omkafka.c

```diff
@@ -743,13 +791,157 @@ deliveryCallback(rd_kafka_t __attribute__((unused)) *rk,
 	} else {
 		DBGPRINTF("omkafka: kafka delivery SUCCESS on msg '%.*s'\n", (int)(rkmessage->len-1),
 			(char*)rkmessage->payload);
+		STATSCOUNTER_INC(ctrKafkaAck, mutCtrKafkaAck);
 	}
 finalize_it:
 	if(iRet != RS_RET_OK) {
 		DBGPRINTF("omkafka: deliveryCallback returned failure %d\n", iRet);
 	}
 }
 
+/**
+ * This function looks for a json object that corresponds to the
+ * passed name and returns it if found. Otherwise returns NULL.
+ * It will be used for processing stats callback json object.
+ */
+static struct fjson_object * get_object(struct fjson_object *fj_obj,
+		const char * name) {
+  struct fjson_object_iterator it = fjson_object_iter_begin(fj_obj);
+  struct fjson_object_iterator itEnd = fjson_object_iter_end(fj_obj);
+  while (!fjson_object_iter_equal (&it, &itEnd)) {
+		const char * key = fjson_object_iter_peek_name (&it);
+		struct fjson_object * val = fjson_object_iter_peek_value(&it);
+		if(!strncmp(key, name, strlen(name))){
+			return val;
+		}
+		fjson_object_iter_next (&it);
+  }
+
+  return NULL;
+}
+
+/**
+ * This function performs a two level search in stats callback json
+ * object. It iterates over broker objects and for each broker object
+ * returns desired level2 value (such as avg/min/max) for specified
+ * level1 window statistic (such as rtt/throttle/int_latency). Threshold
+ * allows skipping values that are too small, so that they don't
+ * impact on aggregate averaged value that is returned.
+ */
+static uint64 jsonExtractWindoStats(struct fjson_object * stats_object,
```

Comment:
we indent by tabs. Please change (one tab = 8 chars standard width)

Reply:
oh, and we follow the Gnu rule of starting the function name on column 1 of the next line